portfolio/src/main/webapp/script.js

```diff
@@ -29,3 +29,9 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact; 
 }
+
```

Comment:
add JSDoc to describe the function: https://google.github.io/styleguide/jsguide.html#jsdoc